releases/v0.5.0.md

```diff
@@ -12,20 +17,22 @@ definition of the acronyms used in this document.
 - Process-based isolation is now fully supported with SGX hardware added to enclave-cc CI
 - [KBS Resource URI Scheme](https://github.com/confidential-containers/attestation-agent/blob/main/docs/KBS_URI.md) 
 is published to identify all confidential resources.
-- SEV configuration can be set with annotations
+- Different KBCs now share image encryption format allowing for interchangeable use.
 - Generic Key Broker System (KBS) is now supported. This includes the [KBS](https://github.com/confidential-containers/kbs)
```

Comment:
```suggestion
- Generic Key Broker System (KBS) is now supported using TDX (support for other TEEs coming soon). This includes the [KBS](https://github.com/confidential-containers/kbs)
```

Reply:
```suggestion
- Generic Key Broker System (KBS) is now supported (With TDX and a sample mode, support for other TEEs is coming soon). This includes the [KBS](https://github.com/confidential-containers/kbs)
```

Reply:
My change differs slightly from @dcmiddle suggestion and it mentions the sample mode.